Sources/Commands/SwiftTool.swift

```diff
@@ -340,14 +337,16 @@ public class SwiftTool {
                 try ProcessEnv.chdir(packagePath)
             }
 
-            // Force building with the native build system on other platforms than macOS.
-#if !os(macOS)
-            self.options._buildSystem = .native
-#endif
-
             let processSet = ProcessSet()
             let buildSystemRef = BuildSystemRef()
-            interruptHandler = try InterruptHandler {
+
+            // trap SIGINT to terminate sub-processes, etc
+            signal(SIGINT, SIG_IGN)
```

Comment:
I think that this is going to break - this needs to be in a `!os(Windows)` block.
```suggestion
#if !os(WIndows)
            signal(SIGINT, SIG_IGN)
#endif
```

Reply:
@compnerd does Windows n require calling `signal()`or the moral equivalent?